apps/table.js

```diff
@@ -997,3 +1014,66 @@ function filterSlides() {
   resetTable();
   pageIndicatorVisible(newSlideRows.length);
 }
+
+function showCollaborationModal (event) {
+  const element = event;
+  const slideId = element.getAttribute('data-slideId');
+  const store = new Store('../data/');
+  store.getSlideCollabDetails(slideId).then(response => {
+    document.getElementById('modal-collab-slidename').innerText = response[0].roomId;
```

Comment:
@Vedant1202 yeah here. It should be slide name. No?

Reply:
Yeah, but not all slides are having names. I guess I'll add a check else return slide Id